src/lib/textMagic.js

```diff
@@ -0,0 +1,26 @@
+const sendTextMagic = async (message) =>
+{
+  const axios = require("axios");
+  const settings = require("../config");
+  var postData = {
+    'text': message,
+    'phones': settings.textMagicData.phoneNumber
+  };
+
+  let axiosConfig = {
+    headers: {
+        'X-TM-Username': settings.textMagicData.username,
+        'X-TM-Key': settings.textMagicData.APIkey
+    }
+  };
+
+  axios.post('https://rest.textmagic.com/api/v2/messages', postData, axiosConfig)
```

Comment:
You created an async function (line 1), you don't need to use `.then()`, you can instead use:

`const response = await axios.post('https://rest.textmagic.com/api/v2/messages', postData, axiosConfig)`